config/zones.json

```diff
@@ -449,12 +449,12 @@
       "geothermal": 0,
       "hydro": 177,
       "hydro storage": 1308,
-      "nuclear": 5943,
+      "nuclear": 4936,
```

Comment:
The source you provided in the other PR (IAEA) is showing an operational capacity of 5942 MW?
Please provide some clarity around the big difference here.

Reply:
Doel 3 reactor was permenantly shut down 09.23.2022. Source: https://afcn.fgov.be/fr/actualites/doel-3-arret-en-toute-surete According to the IAEA, its net capacity was 1006 MW, so the new nuclear capacity is now 4936 MW, not 5942 MW. Right now the IAEA’s 5942 MW nuclear capacity for Belgium is outdated too.

Reply:
Then this should be included in the DATA_SOURCES.md file until IAEA is updated.

Reply:
Correct, but at the time I made the change, Belgium’s Federal Agency for Nuclear Control hadn’t issued this official statement. So I will include this source too.

Reply:
New source now included in the other PR.